src/pages/RFOX/components/Stake/Stake.tsx

```diff
@@ -87,6 +87,7 @@ export const StakeRoutes: React.FC<StakeRouteProps> = ({ headerComponent }) => {
       : undefined,
     stakingAssetId,
   })
+
```

Comment:
ser

Reply:
removed the diff in [bac0207](https://github.com/shapeshift/web/pull/7421/commits/bac0207cf9dfa4c819b0bbdb7c8e4fe82bd2c8ec)